Cargo.toml

```diff
@@ -12,14 +12,16 @@ readme = "README.md"
 edition = "2018"
 
 [features]
+# when the default feature set is updated, verify that the `--features` flags in
+# `.github/workflows/ci.yaml` are updated accordingly
 default = ["native-client", "middleware-logger", "encoding"]
-h1-client = ["async-h1", "wasm-client"]
+h1-client = ["wasm-client", "http-client/h1_client"]
 native-client = ["curl-client", "wasm-client", "http-client/native_client"]
-hyper-client = ["hyper", "hyper-tls", "native-tls", "runtime", "runtime-raw", "runtime-tokio" ]
-curl-client = ["isahc"]
-wasm-client = ["js-sys", "web-sys", "wasm-bindgen", "wasm-bindgen-futures"]
+curl-client = ["http-client/curl_client"]
+wasm-client = ["web-sys", "http-client/wasm_client"]
 middleware-logger = []
-encoding = ["encoding_rs"]
+# requires web-sys for TextDecoder on wasm
+encoding = ["encoding_rs", "web-sys"]
```

Comment:
Won't this also pull in `web-sys` on non-wasm targets? I believe if `wasm-client` has been enabled then `web-sys` is already available.

Reply:
web-sys is still listed as a dependency for wasm targets only—appears to work as expected this way. actually web-sys is no longer directly used by surf for the wasm-client feature, so we can remove it from there